.github/workflows/nightly-tests.yml

```diff
@@ -123,16 +134,20 @@ jobs:
         uses: broadinstitute/action-slack@v3.8.0
         env:
           SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
+          STATUS: failure
```

Comment:
This can only be `failure` but I thought it'd be clearer if the slack parameters were as similar as possible. Maybe we can refactor this somehow, at some point.